data_model/1.1/clusters/BooleanState.xml

```diff
@@ -64,7 +64,7 @@ Davis, CA 95616, USA
   <clusterIds>
     <clusterId id="0x0045" name="Boolean State"/>
   </clusterIds>
-  <classification hierarchy="base" role="application" picsCode="BOOL" scope="Endpoint"/>
+  <classification hierarchy="base" role="application" picsCode="BOOL" scope="Type 2"/>
```

Comment:
Does the spec say Type 2 here? That's...weird. Weird enough that it might cause problems with tests. This might need to be cleaned up in the scripts.